### ISSUE TITLE
`rpc.isNotificationMessage()` missing

### ISSUE THREAD

**Issue by arnoson** [TypeFox/monaco-languageclient#445]
I'm trying to recreate the server side connection forwarding example in the [vscode-ws-jsonrpc readme](https://github.com/TypeFox/monaco-languageclient/tree/main/packages/vscode-ws-jsonrpc):

```ts
import * as rpc from 'vscode-ws-jsonrpc';
import * as server from 'vscode-ws-jsonrpc/server';

const socket: rpc.IWebSocket; // open the web socket
const reader = new rpc.WebSocketMessageReader(socket);
const writer = new rpc.WebSocketMessageWriter(socket);
const socketConnection = server.createConnection(reader, writer, () => socket.dispose())
const serverConnection = server.createServerProcess('Example', 'node', ['example.js']);
server.forward(socketConnection, serverConnection, message => {
    if (rpc.isNotificationMessage(message)) {
        if (message.method === 'testNotification') {
            // handle the test notification
        }
    }
    return message;
});
```

But the method `isNotificationMessage()` doesn't exist in `rpc`. Am I missing something?

**Comment by kaisalmen** [comment]
Hi @arnoson thank you for bringing this up. We forgot to update the README content. The example were updated in the meantime. Take a look at the [server example](https://github.com/TypeFox/monaco-languageclient/blob/main/packages/examples/server/) for now.

I will update the README content and will also add a link to the example where both `monaco-languageclient` and `vscode-ws-jsonrpc` are used.

**Comment by arnoson** [comment]
Awesome, thanks for the quick reply!

**Comment by kaisalmen** [comment]
It is now fixed on main branch, see #446
